bin/reth/src/optimism.rs

```diff
@@ -24,8 +27,20 @@ fn main() {
     }
 
     if let Err(err) = Cli::<RollupArgs>::parse().run(|builder, rollup_args| async move {
-        let NodeHandle { node, node_exit_future } =
-            builder.launch_node(OptimismNode::new(rollup_args.clone())).await?;
+        let NodeHandle { node, node_exit_future } = builder
+            .node(OptimismNode::new(rollup_args.clone()))
+            .extend_rpc_modules(move |ctx| {
+                // register sequencer tx forwarder
+                if let Some(sequencer_http) = rollup_args.sequencer_http.clone() {
+                    ctx.registry.set_eth_raw_transaction_forwarder(Arc::new(SequencerClient::new(
+                        sequencer_http,
+                    )));
+                }
```

Comment:
ah I see, so instead of a cfg'd client arg, there's an optional dyn forwarder, nice